CloneTiving/CloneTiving/Structs&Enums/Content.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Content.swift
+//  CloneTiving
+//
+//  Created by 지희의 MAC on 2023/05/03.
+//
+
+import UIKit
+
+struct Contents {
+    var chanelName: String = "Mnet"
+    var contentName: String
+    var rate: Double
+}
+
+
+
+extension Contents {
+    static func dummy() -> [Contents] {
+        var contents: [Contents] = []
+        let contentNames: [String] = ["보이즈 플래닛 12화","하트시그널 4화","엠카운트다운 230화"]
+        let rates: [Double] = [80.0,24.1,30.2]
+        
+        for number in 0...2 {
+            contents.append(Contents(contentName: contentNames[number], rate: rates[number]))
+        }
```

Comment:
p5;
움 이것도 취향차이 일 수 있겠지만,

```suggestion
        for i in contentNames.count {
            contents.append(Contents(contentName: contentNames[i], rate: rates[i]))
        }
```

요로코롬은 어떨까요??